loris/resolver.py

```diff
@@ -60,12 +60,20 @@ class SimpleFSResolver(_AbstractResolver):
 
     def __init__(self, config):
         super(SimpleFSResolver, self).__init__(config)
-        self.cache_root = self.config['src_img_root']
+        if 'src_img_roots' in self.config:
+            self.cache_roots = self.config['src_img_roots']
+        else:
+            self.cache_roots = [self.config['src_img_root']]
 
-    def is_resolvable(self, ident):
+    def get_fp(self, ident):
         ident = unquote(ident)
-        fp = join(self.cache_root, ident)
-        return exists(fp)
+        for directory in self.cache_roots:
```

Comment:
OK, added some tests and removed the tabs from tests/resolver_t.py.

Don't agree with the comment on changing the loop to a list comprehension, or that it matters in terms of performance in this case. The iteration is through a very small number of directories, and then an os.exists is used.